src/main/java/com/eppo/sdk/dto/BanditCategoricalAttributeCoefficients.java

```diff
@@ -0,0 +1,12 @@
+package com.eppo.sdk.dto;
+
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
```

Comment:
This annotation magically (😞 ) creates getters and setters for each of these private class member.